src/tests/Repository.test.js

```diff
@@ -4,6 +4,8 @@ import Repository from "../components/Repository";
 import {data, match, location} from "./mocks";
 
 describe("<Repository />", () => {
+  const getUserFromJwt = jest.fn();
+  getUserFromJwt.mockReturnValueOnce({id:29888641,login:"filiptronicek"}).mockReturnValue(true);
   it("should render without throwing an error", () => {
     const component = shallow(<Repository location={location} match={match} data={data} />);
```

Comment:
You will need to add the `data.user` here.

```suggestion
    const component = shallow(<Repository user={data.user} location={location} match={match} data={data} />);
```

Reply:
Should I add it just to the first or also the second test? @bdougie

Reply:
yes, it should be both :)

Reply:
Ok, done, I hope it is ok now.